src/main/java/org/testng/TestNG.java

```diff
@@ -1711,6 +1750,11 @@ protected static void validateCommandLineParameters(CommandLineArgs args) {
       throw new ParameterException("Groups option should be used with testclass option");
     }
 
+    if (args.slave != null && args.master != null) {
```

Comment:
the remote package was moved to https://github.com/testng-team/testng-remote in PR #953